lsp-positions/src/lib.rs

```diff
@@ -31,10 +31,19 @@ use std::ops::Range;
 
 use memchr::memchr;
 
+#[cfg(feature = "graphemes")]
+use unicode_segmentation::UnicodeSegmentation as _;
```

Comment:
Is this import to get access to methods in a trait?  If so, as a nit, you should be able to remove the `as _` part.

Reply:
It is indeed to get trait methods into scope. I had seen this done when the trait name itself was not explicitly referenced, and kind of liked it. In that case the trait had a very common name such as `Context`, so greater chance for clashes. I don't feel very strongly either way, I guess.